package.json

```diff
@@ -68,7 +68,7 @@
     "jarvis": "node scripts/jarvis.js",
     "dev-environment-setup": "sh ./scripts/dev-environment-setup.sh",
     "get-affected": "node scripts/print-affected-array.mjs",
-    "symlink:submodules": "nx run-many --target=symlink --all",
+    "symlink:submodules": "pnpm --filter \"@novu/ee-*\" exec node \"$(pwd)/scripts/symlink-ee.mjs\"",
```

Comment:
Calling a single script for these types of operations ensures we only have a single script to modify, reducing maintenance cost.

Reply:
Love the one command. it is true that it will be easier to maintain it unless we will need to maintain symlink-ee script 😅